pipelines/upgrade/05-server_to_intermediate.yaml

```
@@ -21,3 +21,11 @@
     - foreman_server_repositories
     - role: foreman_installer
       foreman_installer_skip_installer: "{{ true if forklift_upgrade_version_start == forklift_upgrade_version_intermediate else false }}"
+  post_tasks:
+    - name: Run remove Pulp 2
+      command: foreman-maintain content remove-pulp2
+      when:
+        - pipeline_version == "4.0"
```

Shouldn't it check the version that's upgraded to instead?
```suggestion
        -  forklift_upgrade_version_intermediate== "4.0"
```